docker-init.php

```diff
@@ -24,3 +24,27 @@
     }
   }
 }
+
+if (!empty(getenv('INIT_COMPANY_COUNTRYCODE'))) {
+    require_once DOL_DOCUMENT_ROOT.'/core/lib/company.lib.php';
+    require_once DOL_DOCUMENT_ROOT.'/core/class/ccountry.class.php';
+    $countryCode = getenv('INIT_COMPANY_COUNTRYCODE');
+    $country = new Ccountry($db);
+    $res = $country->fetch(0,$countryCode);
+    if ($res > 0 ) {
+        $s = $country->id.':'.$country->code.':'.$country->label;
+        dolibarr_set_const($db, "MAIN_INFO_SOCIETE_COUNTRY", $s, 'chaine', 0, '', $conf->entity);
+        printf('Configuring for country : '.$s);
+        activateModulesRequiredByCountry($country->code);
+        $db->commit();
+    }
+    else {
+            printf('Unable to find country '.$countryCode);
+    }
+}
+
+if (!empty(getenv('INIT_COMPANY_NAME'))) {
+    $compname = getenv('INIT_COMPANY_NAME');
+    dolibarr_set_const($db, "MAIN_INFO_SOCIETE_NOM", $compname, 'chaine', 0, '', $conf->entity);
+    $db->commit();
+}
```

Comment:
You forgot to update env name here before running update.sh

Reply:
My bad, sorry.